_docs/setup/kubernetes/quick-start.md

```diff
@@ -93,7 +93,7 @@ $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
 
 Configure `kubectl` CLI based on steps [here](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0/manage_cluster/cfc_cli.html) for how to access the IBM Cloud Private Cluster.
 
-### [OpenShift Origin](https://www.openshift.org) (version 3.7 or later)
+### [OpenShift Origin](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) (version 3.7 or later)
```

Comment:
/cc @hklai 

Should these links be added separately as:
Follow steps here(https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) to install kubernetes on OpenShift Cloud.